### PR TITLE
Bug fix: import data restartability in case of oracle

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

File: yb_migrate/cmd/importData.go [hunk 1]
```diff
@@ -380,8 +379,11 @@ func checkPrimaryKey(tableName string) bool {
 		table = strings.Split(tableName, ".")[0]
 	}
 
+	/* currently object names for yugabytedb is implemented as case-sensitive i.e. lower-case
+	but in case of oracle exported data files(which we use for to extract tablename)
+	so eg. file EMPLOYEE_data.sql -> table EMPLOYEE which needs to converted for using further */
 	checkPKSql := fmt.Sprintf(`SELECT * FROM information_schema.table_constraints
-	WHERE constraint_type = 'PRIMARY KEY' AND table_name = '%s' AND table_schema = '%s';`, table, schema)
+	WHERE constraint_type = 'PRIMARY KEY' AND table_name = '%s' AND table_schema = '%s';`, strings.ToLower(table), schema)
```
Comment (amit-yb) [inline yb_migrate/cmd/importData.go:386]: This will break case-sensitive PG migration.

Comment (sanyamsinghal) [reply yb_migrate/cmd/importData.go]: Just pushed the updated. Have a look @amit-yb

File: yb_migrate/cmd/importData.go [hunk 2]
```diff
@@ -380,6 +379,14 @@ func checkPrimaryKey(tableName string) bool {
 		table = strings.Split(tableName, ".")[0]
 	}
 
+	sourceDBType := ExtractMetaInfo(exportDir).SourceDBType
+	if sourceDBType == ORACLE {
+		table = strings.ToLower(table)
+	}
+
```
Comment (amit-yb) [inline yb_migrate/cmd/importData.go:386]: Can you please confirm the behaviour for MySQL as well?

**Review by amit-yb — APPROVED** [review]
LGTM with an important comment.

File: yb_migrate/cmd/importData.go (hunk 2 above)
Comment (sanyamsinghal) [reply yb_migrate/cmd/importData.go:386]: yes tested that, MySQL exports the `table_data.sql` file into lowercase, so this is not needed.

**Pull request merged**